src/database/modele/Response.js

```diff
@@ -20,14 +22,27 @@ const ResponseSchema = new mongoose.Schema({
     refQuestion:{
         type: Number,
         ref: 'Question',
-        required: true
+        required: true,
+        validate: function(v) {
+            //Validate will permit us to make some validation before a refQuestion is saved
+            //The function check if the id of the value passed exits in the DataBase if yes it will return true otherwise false
```

Comment:
```suggestion
            // Validate will permit us to make some validation before a refQuestion is saved
            // The function check if the id of the value passed exits in the DataBase if yes it will return true otherwise false
```